docs/build/eps.md

```diff
@@ -400,6 +400,33 @@ index 7dfa97c..6d99e71 100644
 Dockerfile instructions are available [here](https://github.com/microsoft/onnxruntime/tree/master/dockerfiles#nuphar).
 
 
+---
+
+## SNPE
+See more information on the SNPE execution provider [here](../execution-providers/SNPE-ExecutionProvider.md).
+
+### Prerequisites
+{: .no_toc }
+* Qualcomm Neural Processing SDK [Linux/Android](https://developer.qualcomm.com/software/qualcomm-neural-processing-sdk)
+* Qualcomm Neural Processing SDK [Windows](https://developer.qualcomm.com/software/qualcomm-neural-processing-sdk/windows-on-snapdragon)
+
+### Build Instructions
+{: .no_toc }
+
+#### Windows
+```
+build.bat --use_snpe --snpe_root=\SNPE\snpe_win_signed-1.61.0 --build_shared_lib --cmake_generator "Visual Studio 16 2019" --skip_submodule_sync --config Release --build_dir \build\Windows_snpe161
```

Comment:
```suggestion
build.bat --use_snpe --snpe_root=[location-of-SNPE_SDK] --build_shared_lib --cmake_generator "Visual Studio 16 2019" --skip_submodule_sync --config Release --build_dir \build\Windows_snpe161
```

Reply:
same for Android and Linux below.

Reply:
I prefer to provide concrete example of a real directory name rather than [location] because it's not always clear which directory/subdirectory we need to point to for a path/root directory.

Reply:
discussed with @hectorsvc , we'll make updates to the build commandline

Reply:
I agree with Manash.
1. SNPE version keeps changing since they have monthly release
2. Here, the version is different with Android, may cause confusion
3. SNPE package is zip in zip, really depend on how user unzip it
My suggestion is use place holder here and state that the path should point to a directory that has SNPE SDK include folder and lib folder
also change --build_dir \build\Windows_snpe161 -> --build_dir \build\Windows

Reply:
yes the key thing is path to lib and include directories.